components/post_view/channel_intro_message/set_header_button.tsx

```diff
@@ -14,15 +15,16 @@ import * as Utils from 'utils/utils';
 
 type Props = {
     channel: Channel;
+    permissions?: string[];
```

Comment:
do we have a union type for the permissions? we could also infer them from the `Permissions` constant

Reply:
permissions is not longer passed into the SetHeaderButton component, so this is not a concern any more.